migrations/133_assessment_access_rules__exam_uuid__add.sql

```diff
@@ -1,2 +1,4 @@
 ALTER TABLE exams ADD COLUMN IF NOT EXISTS uuid uuid UNIQUE;
-ALTER TABLE assessment_access_rules ADD COLUMN exam_uuid uuid;
+ALTER TABLE assessment_access_rules
+    ADD COLUMN exam_uuid uuid,
+    DROP COLUMN exam_id;
```

Comment:
I'm afraid that you're going to need to also delete the column from `database/tables/assessment_access_rules.pg` to keep Travis happy.